### ISSUE TITLE
update scl deprecated tcp/udp

### ISSUE THREAD

**Issue by Kokan** [balabit/syslog-ng#2322]
The `scl` directory contains a few deprecated use of `tcp(...)` and `udp(...)`, they should be replaced with `network( ... transport(tcp/udp) ...)`.

**Comment by mauricetmeyer** [comment]
@Kokan I would like to take on this as part of Hacktoberfest.

**Comment by Kokan** [comment]
@mauricetmeyer sure thing. I saw you already created a commit. In the commit message we follow the rule as the first line should be:
```
<block>: short summary of change

<optional extra message, that do not have rule how to format>
```

By block I mean the part you changed for example: `scl: ...`.

**Comment by mauricetmeyer** [comment]
@Kokan I'll make sure to fix that. Beside that I have a question about an awk script defining a deprecated udp call:

https://github.com/balabit/syslog-ng/blob/66a3519782f08f2e03502d4a5d0e4d3ac3740823/scl/syslogconf/convert-syslogconf.awk#L118

 should I also update that?

**Comment by Kokan** [comment]
@mauricetmeyer if possible yes please (also it has an other one in the same file).

**Comment by mauricetmeyer** [comment]
@Kokan pull request (https://github.com/balabit/syslog-ng/pull/2326) has been made. I actually had the file ready to commit but I wanted to make sure first.

Thank you for being so welcoming!

**Comment by gaborznagy** [comment]
Fixed by #2326
